### ISSUE TITLE
Typescript compilation fails / Cannot find module '@prisma/sdk'

### ISSUE THREAD

**Issue by iherger** [graphql-nexus/nexus-plugin-prisma#499]
Typescript compilation `tsc` using the latest version (0.5.0) fails with the error
```
node_modules/@prisma/photon/dist/utils/getDMMF.d.ts:2:32 - error TS2307: Cannot find module '@prisma/sdk'.
```

**Comment by jasonkuhrt** [comment]
Seems that this should be a prod dep https://github.com/prisma-labs/nexus-prisma/blob/master/package.json#L30

**Comment by iherger** [comment]
Or it should be a prod dep in `@prisma/photon`

**Comment by jasonkuhrt** [comment]
@iherger good point, but in this case I think its designed as a peer dep: https://github.com/prisma/photonjs/blob/master/packages/photon/package.json#L68-L70. @timsuchanek can you confirm this?

Either `nexus-prisma` needs to bring a version of `@prisma/sdk` or the end-user does in their app. I would prefer not concerning the end-user with this.

**Comment by jasonkuhrt** [comment]
Fixed in https://github.com/prisma-labs/nexus-prisma/releases/tag/v0.5.1

**Comment by iherger** [comment]
This doesn't seem to be fixed in version `0.5.1`, see #524 

In addition, when doing `yarn install` I get the following warning:
```
warning "nexus-prisma > @prisma/photon@0.2.152" has unmet peer dependency "@prisma/sdk@*".
```

**Comment by jasonkuhrt** [comment]
Thanks @iherger yeah!
